site/content/events/2015-siliconvalley/proposals/Example_Talk/index.txt

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Remove the Example_Talk directory - it's just intended to show you how to use it.